### DISCUSSION TITLE
AWS creation failed

### DISCUSSION THREAD

**Discussion by weidiguo** [mozilla/hubs#5271] [Hubs Cloud Questions]
hello!

Be badly in need of help
AWS said it was our docking problem
We built it over 20 times

**Comment by weidiguo** [comment]
![微信图片_20220321144642](https://user-images.githubusercontent.com/94850229/159218387-63ba6761-232e-49da-ba72-3288a5fa296c.png)

**Reply by pattersonbl2** [reply]
Was this able to be fixed?

**Comment by weidiguo** [comment]
Hello, honey. We've taken care of that. Thank you very much

发自我的iPhone

**Comment by weidiguo** [comment]
Has been resolved Thank you very much



At 2022-03-23 23:52:22, "Brandon Patterson" ***@***.***> wrote:

Was this able to be fixed?

**Comment by weidiguo** [comment]
Hello, we will not ask for help when we encounter trouble in creating CDN










At 2022-03-25 13:23:33, "谢氏" ***@***.***> wrote:

Has been resolved Thank you very much



At 2022-03-23 23:52:22, "Brandon Patterson" ***@***.***> wrote:

Was this able to be fixed?
